OpenShift/SalesDemoContainer/scripts/ycsb_benchmark/ycsb-0.12.0/couchbase2-binding/README.md

```diff
@@ -0,0 +1,145 @@
+<!--
```

Comment:
It looks like the nuodb-specific stuff for ycsb is in nuo-config. I see the same files in multiple places, and I see stuff for other databases, which we don't need to include in our github.